### PR TITLE
Add footer, add eventcard, add FontAwesome icons, removed folders in …

### PR DESCRIPTION
**Pull request by fre-ben**
…storybook, modified some existing files, add images

### CODE REVIEW

File: src/components/button/button.css [hunk 1]
```diff
@@ -1,8 +1,24 @@
...
+  width: 138px;
+  height: 55px;
+  border-radius: 27pt;
+  background-color: var(--bg-color);
+}
+
+.btn-home-img {
+  color: var(--secondary-color);
+  padding-right: 15px;
+  height: 20px;
```
Comment (m-wiemers) [inline src/components/button/button.css:15]: height is given from picture

```suggestion
  color: var(--secondary-color);
  padding-right: 15px;
```

File: src/components/button/button.css [hunk 2]
```diff
@@ -1,8 +1,24 @@
...
+}
+
+.btn-home-text {
+  color: var(--secondary-color);
+  font-family: var(--primary-font);
+  font-weight: 600;
+  align-self: center;
+  padding-top: 2px;
```
Comment (m-wiemers) [inline src/components/button/button.css:23]: can`t see any changes, when align-self is deleted

```suggestion
  color: var(--secondary-color);
  font-family: var(--primary-font);
  font-weight: 600;
  padding-top: 2px;
```

File: src/components/footer/footer.css [hunk 3]
```diff
@@ -0,0 +1,15 @@
+.footer-grid {
+  display: grid;
+  grid-auto-flow: column;
+  justify-content: center;
+  align-content: center;
+  column-gap: 69px;
+  background-color: var(--footer-color);
+  height: 98px;
+  width: 100vw;
+}
```
Comment (m-wiemers) [inline src/components/footer/footer.css:10]: cant see any changes when width is deleted

```suggestion
  column-gap: 69px;
  background-color: var(--footer-color);
  height: 98px;
}
```

File: src/components/events/events.css [hunk 4]
```diff
@@ -0,0 +1,48 @@
+.eventcard-grid {
+  display: grid;
+  grid-auto-flow: column;
+  border-radius: 10pt;
+  background-color: var(--events-bg-color);
+  width: 364px;
+  height: 98px;
+}
```
Comment (m-wiemers) [inline src/components/events/events.css:8]: There is a problem when small mobile is selected. With grid-template-columns we don't need a fixed width anymore and the problem is solved.

```suggestion
.eventcard-grid {
  display: grid;
  grid-auto-flow: column;
  grid-template-columns: 2fr 1fr;
  border-radius: 10pt;
  background-color: var(--events-bg-color);
}
```

File: src/components/events/events.css [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+.eventcard-left {
+  display: grid;
+  grid-auto-flow: row;
+  justify-content: start;
+  align-content: center;
+  padding-left: 20px;
+}
```
Comment (m-wiemers) [inline src/components/events/events.css:16]: Cant see any changes, when justify-content is deleted ;-)

```suggestion
.eventcard-left {
  display: grid;
  grid-auto-flow: row;
  
  align-content: center;
  padding-left: 20px;
}
```

**Review by m-wiemers — APPROVED** [review]
Very good job! 🚀

File: src/components/button/button.css (hunk 1 above)
Comment (fre-ben) [reply src/components/button/button.css:15]: It's actually a small change, barely noticeable.
The height value is taken from the XD file. It's a really small difference, wouldn't make a real difference, I think

File: src/components/button/button.css (hunk 2 above)
Comment (fre-ben) [reply src/components/button/button.css:23]: Yup! Deleted it :)

File: src/components/footer/footer.css (hunk 3 above)
Comment (fre-ben) [reply src/components/footer/footer.css:10]: True, thanks!

File: src/components/events/events.css (hunk 4 above)
Comment (fre-ben) [reply src/components/events/events.css:8]: Great change for responsiveness! Didn't really get that. 🚀

File: src/components/events/events.css (hunk 5 above)
Comment (fre-ben) [reply src/components/events/events.css:16]: True, I think thats the default value, so no ned to specify it. Thanks!

**Review by m-wiemers — APPROVED** [review]

**Pull request merged**